UI/src/utils/backend-utils.js

```diff
@@ -112,13 +112,18 @@
       });
     }, 1000);
 
-    this.getAssets = function (boundingBox, filter) {
+    this.getAssets = function (boundingBox, filter, checkbox) {
       if(!filter)
         filter = function(assets){return assets;};
 
-      self.getAssetsWithCallback(boundingBox, function (assets) {
+      var assetsFetched = function (assets) {
         eventbus.trigger('assets:fetched',filter(assets));
-      });
+      };
+
+      if(checkbox){
+        self.getAssetsWithCallbackServiceStops(boundingBox, assetsFetched);
+      }
+      self.getAssetsWithCallback(boundingBox, assetsFetched);
```

Comment:
Can we modify to something like that:
```javascript
val serviceToCall = checkbox ? self.getAssetsWithCallbackServiceStops : self.getAssetsWithCallback;

serviceToCall(boundingBox, assetsFetched);

Reply:
No! The idea here is to call both of them if checkbox is checked! If not checked call only the bus stop one

Reply:
Ok, you are right.
Done without any modification.